gammapy/makers/utils.py

```diff
@@ -441,6 +441,10 @@ def make_theta_squared_table(
     position_off : `astropy.coordinates.SkyCoord`
         Position from which the OFF theta^2 distribution is computed.
         Default is reflected position w.r.t. to the pointing position.
+    energy_edges : list of `~astropy.units.Quantity`, optional
+        Edges of the energy bin where the theta squared distribution
+        is evaluated. For now, only one interval is accepted.
```

Comment:
Thanks @fabiopintore does it fails if there is more than one interval and what is the error message ?

Reply:
Thanks @QRemy . I've added an error